baremaps-postgres/src/test/java/com/baremaps/postgres/PostgisPluginTest.java

```diff
@@ -75,4 +72,90 @@ void test() {
     assertEquals(record.getLineString(), result.get(0).getLineString());
     assertEquals(record.getPolygon(), result.get(0).getPolygon());
   }
+
+  @Test
+  @Tag("integration")
+  void beanMapper() {
+    var id = "1";
+    var type = "Feature";
+    var geometry = new GeometryFactory().createPoint(new Coordinate(1, 1));
+    var properties = Map.of("k1", "v1", "k2", "v2");
+    var feature = new Feature(id, type, geometry, properties);
+
+    Jdbi jdbi = Jdbi.create("jdbc:tc:postgis:13-3.1:///test").installPlugin(new PostgisPlugin());
```

Comment:
Maybe use the `com.baremaps.testing.TestConstants.DATABASE_URL` ? instead of hard coded String.